pgfinder/pgio.py

```diff
@@ -23,6 +24,29 @@
 LOGGER = logging.getLogger(LOGGER_NAME)
 
 
+# FIXME: Needs type and docstring updated to dict
```

Comment:
This would be a quick and easy fix to include here (PyLint usually complains about `FIXME` and `TODO` comments too).

Reply:
Agreed! Done now!